### ISSUE TITLE
Clarification on output 

### ISSUE THREAD

**Issue by jeffmylife** [calico/basenji#16]
The output array for basenji is: (960, 4229)
Could you explain in more detail where the 960 comes from? I understand that it is the number of bins on the sequence, but why is it not 1024bp bins? 1024 comes from 2^17/128 = 1024.

Additionally, in the picture below there is a labeling ("pred/n/m") that is ordered by digit rather than by value. Which ordering should we consult when looking at [Supplementary Table 1](https://www.biorxiv.org/content/early/2018/03/22/161851.figures-only)?  I presume the 10th output feature corresponds to the 10th value in SuppTable1, not the third (as it would under digit order). 

![screen shot 2018-06-01 at 4 26 01 pm](https://user-images.githubusercontent.com/25440288/40915095-c03430de-67c8-11e8-9d14-38a01cec0a93.png)

Thanks

**Comment by davek44** [comment]
Hi Jeff,

The training code takes a parameter called "batch_buffer" set to 4096 for these models that tells the loss function to ignore the 4096 bp on each side. The 960 length is taken after slicing these positions out.

The target order won't match that in the supplementary table. Where are you getting this model from? I'm not currently saving the target labels with the model; I usually have the training dataset nearby.

Best,
Dave

**Comment by jeffmylife** [comment]
That makes sense, thank you. 
I got it from [kipoi](http://kipoi.org/), a genomics model zoo that came out around March. Specifically, [here](https://github.com/kipoi/models). 
If the output doesn't match the supplementary table, could you provide a mapping?

Jeff

**Comment by davek44** [comment]
Got ya. Then it should refer to https://storage.googleapis.com/131k/sample_wigs.txt as specified in https://github.com/calico/basenji/blob/master/manuscript/get_data.sh.

**Comment by jeffmylife** [comment]
Cool, i'll direct others using kipoi to do the same. 

Much appreciated, 
Jeff
